src/networking/controllers/ToDoController.js

```diff
@@ -0,0 +1,56 @@
+import HttpService from '../HttpService';
+import toDoStore from '../../scenes/app/stores';
+import Utils from '../../helpers/Utils';
+
+class ToDoController {
+  //TODO: change the way of handle errors and remove the console logs.
+  getToDo = async () => {
+    try {
+      const response = await HttpService.get('');
+      toDoStore.addItems(response.data);
+    } catch (error) {
+      console.log(error);
+    }
+  }
+
+  sendToDo = async (newToDo) => {
+    try {
+      const response = await HttpService.post('', newToDo);
+      const result = toDoStore.addToDo(response.data);
+      return result;
+    } catch (error) {
+      console.log(error);
+      return false;
+    }
+  }
+
+  deleteToDo = async (toDo) => {
+    try {
+      const id = Utils.getId(toDo.url);
+      const response = await HttpService.delete(id);
+      toDoStore.removeToDo(response.data);
+    } catch (error) {
+      console.log(error);
+    }
+  }
+
+  patchToDo = async (id) => {
+    try {
+      const body = toDoStore.getToDoById(id);
+      body.completed = !body.completed;
+      const response = await HttpService.patch(id, body);
+      toDoStore.updateToDo(response.data);
+    } catch (error) {
+      console.log(error);
+    }
+  }
+
+  clearAllDone = () => {
```

Comment:
It seems to me that this method should be inside `toDoStore`. A controller's responsibility is communication with an API or service. It should be agnostic as to what this data means or what it is used for.